gce/survival-training/dummy/train.py

```diff
@@ -0,0 +1,219 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
```

Comment:
Hate to bring it up now (should have mentioned sooner) but the word survival is ambiguous. Its not clear survival against what? May be rename these to fault-tolerant, fail-safe, fail-secure etc something along those lines. Also because these are more standard terms and evoke certain thoughts about what it might be, than survival-training.